cmd/manager/mock.go

```diff
@@ -0,0 +1,73 @@
+package manager
```

Comment:
This implementation is not needed. We have mock for Manager interface built by [mockery](https://github.com/Dynatrace/dynatrace-operator/blob/main/test/mocks/sigs.k8s.io/controller-runtime/pkg/manager/manager.go)

Reply:
removed in [8d47858](https://github.com/Dynatrace/dynatrace-operator/pull/2289/commits/8d4785872b1570310f4f1f3e084a692d52e52202)